File: LeetCode Problems/1501 to 1600/1502. Can Make Arithmetic Progression From Sequence/1502. Can Make Arithmetic Progression From Sequence.cpp
class Solution {
public:
    bool canMakeArithmeticProgression(vector<int> &arr) {
        sort(arr.begin(), arr.end());
        int temp = arr[1] - arr[0];
        for (int i = 1; i < arr.size() - 1; i++) {
            if (arr[i + 1] - arr[i] != temp)
                return false;
        }
        return true;
    }
};

File: LeetCode Problems/1501 to 1600/1502. Can Make Arithmetic Progression From Sequence/1502. Can Make Arithmetic Progression From Sequence.java
class Solution {
    public boolean canMakeArithmeticProgression(int[] arr) {
        Arrays.sort(arr);
        int temp = arr[1] - arr[0];
        for (int i = 1; i < arr.length - 1; i++) {
            if (arr[i + 1] - arr[i] != temp) return false;
        }
        return true;
    }
}
